src/Logic.hs

```diff
@@ -567,7 +570,7 @@ provideFeedback :: ProjectState -> Action ProjectState
 provideFeedback state
   = foldM (flip leaveFeedback) state
   $ filter (Pr.needsFeedback . snd)
-  $ fmap (\(key, pr) -> (PullRequestId key, pr))
+  $ fmap (first PullRequestId)
```

Comment:
Nit: the explicit lambda was somewhat more informative

Reply:
I beg to differ - I am not really fond of this _temp_ variables with hastily made-up names :smile: